aws/cloudtrail.go

```diff
@@ -0,0 +1,64 @@
+package aws
+
+import (
+	"time"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/aws/aws-sdk-go/service/cloudtrail"
+	"github.com/gruntwork-io/cloud-nuke/config"
+	"github.com/gruntwork-io/cloud-nuke/logging"
+	"github.com/gruntwork-io/go-commons/errors"
+)
+
+func getAllCloudtrailTrails(session *session.Session, excludeAfter time.Time, configObj config.Config) ([]*string, error) {
```

Comment:
Noticed that arguments `excludeAfter` `configObj` aren't used which means that will not be possible to filter CloudTrail by `include / exclude` configurations

Reply:
Yes - this is correct. I didn't implement support for it currently because I haven't yet been able to find a reasonable way to get a given Trail's creation date, which I believe rules out the `excludeAfter` pattern. 

Support for the `configObj` pattern would be pretty important for folks who want to avoid nuking an important Trail - so let me see if I can work that into this PR.

Reply:
Implemented support for the `configObj` pattern in 4cb1460.